core/pubnub_ccore_pubsub.c

```diff
@@ -133,20 +133,20 @@ char const* pbcc_get_msg(struct pbcc_context* pb)
                 free(trimmed);
 
                 if (NULL == encrypted.ptr) {
-                    PUBNUB_LOG_ERROR("pbcc_get_msg(pbcc=%p) - base64 decoding failed. Dropping message!\n", pb);
-                    return NULL;
+                    PUBNUB_LOG_WARNING("pbcc_get_msg(pbcc=%p) - base64 decoding failed. Returning original message!\n", pb);
+                    return rslt;
                 }
 
                 pubnub_bymebl_t rslt_block = pb->crypto_module->decrypt(pb->crypto_module, encrypted);
                 free(encrypted.ptr);
                 if (NULL == rslt_block.ptr) {
-                    PUBNUB_LOG_ERROR("pbcc_get_msg(pbcc=%p) - decryption failed. Dropping message!\n", pb);
-                    return NULL;
+                    PUBNUB_LOG_WARNING("pbcc_get_msg(pbcc=%p) - decryption failed. Returning original message!\n", pb);
+                    return rslt;
                 }
 
                 if (pb->decrypted_message_count >= PUBNUB_MAX_DECRYPTED_MESSAGES) {
-                    PUBNUB_LOG_ERROR("pbcc_get_msg(pbcc=%p) - maximum number of decrypted messages reached. Dropping message!\n", pb);
-                    return NULL;
```

Comment:
The reason is the same above. Probably the original implementation is correct one.

Reply:
as above